waltz-ng/client/common/services/index.js

```diff
@@ -74,10 +74,10 @@ export default (module) => {
     iconNameService.register('usageKind', usageKindIconNames);
 
     module.run([
-        'DataTypesDataService',
-        (DataTypesDataService) =>
-            DataTypesDataService
-                .findAll()
+        'DataTypesService',
+        (DataTypesService) =>
```

Comment:
not needed for this pull req - but should probably move to explicit injection style:

````
function runner(dataTypesDataService) {
 ...
}

runner.inject = ['DataTypesDataService' ];

module.run(runner);

```